### PR TITLE
feat(MeetingsSdkAdapter): move video control in a separate file

### PR DESCRIPTION
**Pull request by patricia0817**

### CODE REVIEW

**Comment by patricia0817** [comment]
Task link: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-252488

File: src/MeetingsSDKAdapter.js [hunk 1]
```diff
@@ -24,6 +23,7 @@ import {
 import AudioControl from './MeetingsSDKAdapter/controls/AudioControl';
 import ExitControl from './MeetingsSDKAdapter/controls/ExitControl';
 import JoinControl from './MeetingsSDKAdapter/controls/JoinControl';
+import VideoControl from './MeetingsSDKAdapter/controls/VideoControl';
```
Comment (cipak) [inline src/MeetingsSDKAdapter.js:26]: alpha order

File: src/MeetingsSDKAdapter.js [hunk 2]
```diff
@@ -668,88 +664,45 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
...
-        state = MeetingControlState.DISABLED;
-      } else if (videoEnabled) {
-        // Mute the video only if there is an active meeting
-        if (isInSession) {
-          await sdkMeeting.muteVideo();
-        }
+      await this.updateMeeting(ID, async (meeting, sdkMeeting) => {
+        const isInSession = !!meeting.remoteVideo;
+        const videoEnabled = !!meeting.localVideo.stream;
+        const videoDisabled = !!this.meetings[ID].disabledLocalVideo;
```
Comment (cipak) [inline src/MeetingsSDKAdapter.js:671]: use `meeting` instead of `this.meetings[ID]`

File: src/MeetingsSDKAdapter/controls/VideoControl.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+   * @param {string} meetingID  Id of the meeting to mute video
+   */
+  action(meetingID) {
+    return this.adapter.handleLocalVideo(meetingID);
+  }
+
+  /**
+   * Returns an observable that emits the display data of a mute meeting video control.
+   *
+   * @param {string} meetingID  ID of the meeting to mute video
```
Comment (cipak) [inline src/MeetingsSDKAdapter/controls/VideoControl.js:26]: just "Meeting id", we are not muting here

**Review by cipak — APPROVED** [review]

**Pull request merged**